test/unit/sth_test.js

```diff
@@ -969,6 +969,110 @@ describe('Data removal', function() {
   );
 });
 
+describe('POST /admin/log', function() {
```

Comment:
In fact, this is something that went out of hands in the case of the STH component :) and I thought that for better readability of the tests (and since they are tests) I preferred to opt for this long version... :) Anyhow, I do agree with your suggestion and I'll try to keep it under control to avoid the current situation of the STH component... :) Thanks!